include/veriblock/bfi/bitcoin/block.hpp

```diff
@@ -34,6 +38,65 @@ struct BlockHeader : public BtcBlock {
   }
 };
 
+struct Block : public BlockHeader {
+  std::vector<Transaction> vtx{};
+  PopData popData{};
```

Comment:
remove this. Block is a BTC block, not BTCSQ

Reply:
Ops, forget it :)